virtual_undercloud.yml

```diff
@@ -1,6 +1,11 @@
 ---
 - hosts: undercloud_hypervisor
   tasks:
+    - name: Clean network interfaces
+      shell: |
+        /root/clean-interfaces.sh --nuke
+      changed_when: false
```

Comment:
Will it throw error if clean-interfaces.sh not present on the hypervisor, for example, CI machine?

Reply:
Fixed it